OpenTabletDriver.UI.Library/Controls/NavigationPaneView.axaml

```diff
@@ -0,0 +1,51 @@
+﻿<UserControl
+    xmlns="https://github.com/avaloniaui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:vm="using:OpenTabletDriver.UI.ViewModels"
+    mc:Ignorable="d"
+    d:DesignWidth="800"
+    d:DesignHeight="450"
+    x:Class="OpenTabletDriver.UI.Controls.NavigationPaneView"
+    x:DataType="vm:NavigationPaneViewModel"
+    x:CompileBindings="True">
+  <UserControl.Styles>
+    <Style Selector="TextBlock">
+      <Setter Property="FontSize" Value="14" />
```

Comment:
add fontsize to build.props

Reply:
Same case as above, can extract it into a style/resource instead.

Reply:
Resolved